sync/class.jetpack-sync-actions.php

```diff
@@ -165,85 +168,19 @@ static function schedule_initial_sync( $new_version = null, $old_version = null
 			$initial_sync_config['users'] = 'initial';
 		}
 
-		// we need this function call here because we have to run this function
-		// reeeeally early in init, before WP_CRON_LOCK_TIMEOUT is defined.
-		wp_functionality_constants();
-
-		if ( is_multisite() ) {
-			// stagger initial syncs for multisite blogs so they don't all pile on top of each other
-			$time_offset = ( rand() / getrandmax() ) * self::INITIAL_SYNC_MULTISITE_INTERVAL * get_blog_count();
```

Comment:
Since we are deleting the `INITIAL_SYNC_MULTISITE_INTERVAL` we should also re move it every where in the file.